learning_examples_py/migrations_cpswap/listen_logsubscribe.py

```
@@ -0,0 +1,131 @@
+            max_supported_transaction_version=0,
+        )
+        
+        if not resp.value:
+            print(f"[ERROR] Transaction not found: {signature}")
+            return
+        
+    instructions = resp.value.transaction.transaction.message.instructions
+    
+    for instruction in instructions:
+        if instruction.program_id == RAYDIUM_LAUNCHPAD_PROGRAM_ID and instruction.data == "PotQtwz6wf1":
+            if len(instruction.accounts) == 38:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)